backend/LexCore/Auth/LexAuthUser.cs

```diff
@@ -93,7 +93,7 @@ public LexAuthUser(User user)
             ? Array.Empty<AuthUserProject>() // admins have access to all projects, so we don't include them to prevent going over the jwt limit
             : user.Projects.Select(p => new AuthUserProject(p.Role, p.ProjectId)).ToArray();
         EmailVerificationRequired = user.EmailVerified ? null : true;
-        CanCreateProjects = user.CanCreateProjects ? true : null;
+        CanCreateProjects = user.CanCreateProjects || user.IsAdmin ? true : null;
```

Comment:
instead of doing this in the jwt should we do it client side when parsing the jwt? Just because our jwt budget is limited.

Reply:
I like that idea. We don't really have to worry about an admin's jwt budget, because they don't have any projects, but...doing it client-side means we don't have to wait for new JWT's to roll out 😆.
I'll do that 👍